easybuild/easyconfigs/o/orjson/orjson-3.9.15-GCCcore-12.3.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'orjson'
+version = '3.9.15'
+
+homepage = 'https://github.com/ijl/orjson'
+description = """Fast, correct Python JSON library supporting dataclasses, datetimes, and numpy."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
```

Ah, now the CI is complaining that it needs GCC or binutils as (build) dep, we can fix that by doing:
```suggestion
builddependencies = [
    ('binutils', '2.40'),
```